dbus/dbus.go

```diff
@@ -64,12 +62,6 @@ type MethodCallCallback = func(
 	parameters string,
 ) (interface{}, error)
 
-// TokenAndServerURL stores values for the JWT token and the server URL
-type TokenAndServerURL struct {
-	Token     string
-	ServerURL string
-}
-
 // GetDBusAPI returns the global DBusAPI object
 func GetDBusAPI() (DBusAPI, error) {
```

Comment:
This is very suspicious: we should not need anything from D-Bus in `mender-setup`. This tool is an standalone tool to just write a config file.

Please investigate from where these methods are called and we most likely can remove it from there instead.

Side note: this would be nice to be on a separate commit, as it actually would remove some functionality from the existing code base.